### PR TITLE
Add projects to graphql query root

### PR DESCRIPTION
**Pull request by marten**
# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: app/graphql/types/api_user_type.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module Types
+  class ApiUserType < GraphQL::Schema::Object
```
Comment (camallen) [inline app/graphql/types/api_user_type.rb]: Any reason this isn't a BaseObject?
```suggestion
   class ApiUserType < BaseObject
```

**Comment by marten** [comment]
Yes, and in all of the schema, so you also have to request `launchRequested` and the docs in Graphiql show that the field is called `launchRequested`. In the old code we already did this ourselves: `field :launchRequested, !types.Boolean, property: :launch_requested `. Now that can just be `field :launch_requested, Boolean, null: false`.

File: app/graphql/graphql_schema.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
-GraphqlSchema = GraphQL::Schema.define do
-  query(QueryRoot)
+class GraphqlSchema < GraphQL::Schema
+  query(Types::QueryType)
+  # mutation(Types::MutationType)
```
Comment (camallen) [inline app/graphql/graphql_schema.rb]: do we need this comment?

File: app/graphql/types/mutation_type.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module Types
+  class MutationType < Types::BaseObject
+    # TODO: remove me
```
Comment (camallen) [inline app/graphql/types/mutation_type.rb]: is this still TODO?

File: app/graphql/types/project_type.rb [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    field :primary_language, String, null: true
+
+    field :href, String, null: true
+    field :tags, [String], null: true
+
+    field :completeness, Float, null: false
+    field :activity, Integer, null: false
+    field :researcher_quote, String, null: true
+    field :mobile_friendly, Boolean, null: false
+    # field :urls, [ProjectUrlType]
```
Comment (camallen) [inline app/graphql/types/project_type.rb]: we need these?

File: app/policies/project_policy.rb [hunk 5]
```diff
@@ -5,6 +5,13 @@ class ReadScope < Scope
     def public_scope
       scope.where(private: false)
     end
+
+    def resolve(action =nil)
+      puts action
```
Comment (camallen) [inline app/policies/project_policy.rb]: do we want these debug statements wrapping a super?

**Pull request merged**